docs/automated-workflows/dagman-introduction.rst

```diff
@@ -50,16 +51,18 @@ of parents and children.
 Example: Diamond DAG
 --------------------
 
-A simple diamond-shaped DAG, as shown in the following image is presented as a
-starting point for examples. This diamond DAG contains 4 nodes.
+.. sidebar:: Diamond DAG
```

Comment:
Should we have a convention where we put example file names in the caption, like "example diamond.dag" file?

Reply:
Discussed: It would probably be beneficial if all `.. code-block:: type` that represent a file use the `:caption: test` tag to specify a filename to be used as an identifier for reference in other parts of a documentation page.